src/radical/entk/task.py

```diff
@@ -928,13 +940,30 @@ def path(self, value):
     @tag.setter
     def tag(self, value):
 
+        warnings.warn("Attribute tag will be depcrecated", DeprecationWarning)
+
+        # this method exists for backward compatibility
         if not isinstance(value, str):
             raise ree.TypeError(entity='tag', expected_type=str,
                                 actual_type=type(value))
-
         self._tag = value
```

Comment:
Shouldn't that line become
```py
self._tags = {'colocate': value}
```
?

Reply:
Now that you mention it. I can have the tags work as intended, translate them so that RP understands, and later update them to the new style of tags. Let me do it.

Reply:
Fixed